src/examples/forms/radio/Hidden.tsx

```diff
@@ -0,0 +1,44 @@
+/**
+ * Copyright Zendesk, Inc.
+ *
+ * Use of this source code is governed under the Apache License, Version 2.0
+ * found at http://www.apache.org/licenses/LICENSE-2.0.
+ */
+
+import React, { useState } from 'react';
+import { Field, Label, Radio } from '@zendeskgarden/react-forms';
+
+const Example = () => {
+  const [radioValue, setRadioValue] = useState('annual');
+
+  return (
+    <div role="group" aria-label="Choose a plant lifecycle">
+      <Field>
+        <Radio
+          name="hidden-example"
+          value="annual"
+          checked={radioValue === 'annual'}
+          onChange={event => setRadioValue(event.target.value)}
+        >
+          <Label hidden style={{ display: 'inline-block' }}>
+            Annual
+          </Label>
+        </Radio>
+      </Field>
+      <Field>
+        <Radio
+          name="hidden-example"
+          value="perennial"
+          checked={radioValue === 'perennial'}
+          onChange={event => setRadioValue(event.target.value)}
+        >
+          <Label hidden style={{ display: 'inline-block' }}>
```

Comment:
Display inline-block is [required to display input on hidden label](https://github.com/zendeskgarden/react-components/blob/master/packages/forms/src/styled/radio/StyledRadioLabel.ts#L35
). Suggestions welcomed.

Reply:
Please remove `style` – you're duplicating built-in styling.